
# Part description 

string name
string type
string id
geometry_msgs/Pose current_pose
